CAM/CreateCam.md

```diff
@@ -7,23 +7,23 @@ ns: CAM
 // 0xC3981DCE61D9E13F 0xE9BF2A7D
 Cam CREATE_CAM(char* camName, BOOL active);
 ```
-Create a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.
+Creates a camera with the specified cam name, You can use `SET_CAM_` natives to manipulate the camera.  
+Make sure to call [RENDER_SCRIPT_CAMS](#_0x07E5B515DB0636FC) once the camera is created, or this won't have any visible effect.
+
+**Camera names:**  
+- DEFAULT_SCRIPTED_CAMERA 
```

Comment:
Technically, there's more in `cameras.#mt`. I'm not sure if any have obvious names, but it'd be useful to check for any script cameras in there.

Reply:
Will look into that, thanks